protocol-units/da/m1/setup/src/common/mod.rs

```diff
@@ -0,0 +1,3 @@
+pub mod celestia;
```

Comment:
Do these modules all need to be available to the entire workspace? If not maybe `pub(crate) mod celestia` is a bit tighter.